cypress/support/utils/pricelist.js

```diff
@@ -80,7 +80,6 @@ export class PriceList {
     describe(`Create new PriceListVersion ${priceListVersion.name}`, function() {
       cy.selectTab('M_PriceList_Version');
       cy.pressAddNewButton();
-      cy.writeIntoStringField('Name', `${priceListVersion.name} ${priceListVersion.validFrom}`, true, null, true);
       cy.writeIntoStringField('ValidFrom', priceListVersion.validFrom, true, null, true);
```

Comment:
I have just tried to add a name and it works, it isn't overriden: 
![image](https://user-images.githubusercontent.com/50820854/62774897-dad55280-baae-11e9-9d77-ecc6e64477a2.png). Let's have a chat on Monday.

Reply:
The issue is that in Product Prices, or when filtering by a PLV the "autogenerated" name is used, therefore what is set manually as PLV name is useless imo. eg.: in the picture i could have named my PLV "PricelistFor<test name here> 1", and also have a second one  "PricelistFor<test name here> 2", but that would not be seen here in the selection dialog:
![image](https://user-images.githubusercontent.com/4482210/62847270-0a1dd680-bcde-11e9-8dfd-2fed669be993.png)